packages/react-composites/tests/browser/call/hermetic/UserFacingDiagnostics.test.ts

```diff
@@ -31,6 +31,66 @@ test.describe('User Facing Diagnostics tests', async () => {
     expect(await stableScreenshot(page)).toMatchSnapshot('tile-when-ufd-network-reconnect-is-bad.png');
   });
 
+  /* @conditional-compile-remove(teams-meeting-conference) */
+  test('Teams meeting phone info tile should be showing when network reconnect is bad ', async ({
+    page,
+    serverUrl
+  }) => {
+    const initialState = defaultMockCallAdapterState();
+    initialState.isTeamsMeeting = true;
+    if (initialState.call) {
+      initialState.call.teamsMeetingConference = [
+        {
+          phoneNumber: '1234567890',
+          conferenceId: '',
+          isTollFree: false
+        }
+      ];
+    }
+    setNetworkDiagnostic(initialState, {
+      networkReconnect: { value: DiagnosticQuality.Bad, valueType: 'DiagnosticQuality' }
+    });
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
+
+    await waitForSelector(page, dataUiId('call-composite-hangup-button'));
+    expect(await stableScreenshot(page)).toMatchSnapshot('tile-when-teams-tile-when-ufd-network-reconnect-is-bad.png');
+  });
+
+  test('Tile notification in case of bad network connection ', async ({ page, serverUrl }) => {
```

Comment:
do we require this test?

Reply:
It is. Teams meeting and generic acs call will have difference notifications becuase for acs we have no phone info